engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1003,6 +1002,15 @@ struct divine_toll_t : public paladin_spell_t
       p() -> active.divine_toll -> schedule_execute();
     }
   }
+
+  void execute() override
+  {
+    paladin_spell_t::execute();
+    if ( p() -> conduit.ringing_clarity -> ok() && rng().roll( p() -> conduit.ringing_clarity.percent() ) )
+      for ( int hits = 0; hits < p() -> conduit.ringing_clarity -> effectN( 2 ).base_value(); hits ++ )
+        p() -> active.divine_toll -> schedule_execute();
```

Comment:
Might want to make sure that this uses the right target, as the tooltip mentions "on the main target".